server/service/src/sync/sync_serde.rs

```diff
@@ -59,5 +59,6 @@ where
 /// change on our side.
 pub fn naive_time<'de, D: Deserializer<'de>>(d: D) -> Result<NaiveTime, D::Error> {
     let secs = u32::deserialize(d)?;
-    Ok(NaiveTime::from_num_seconds_from_midnight(secs, 0))
+    Ok(NaiveTime::from_num_seconds_from_midnight_opt(secs, 0)
+        .unwrap_or(NaiveTime::from_hms(0, 0, 0)))
```

Comment:
What was the problem with the input data? number of seconds > 24h? should we fix it manually or is the input just broken?

Reply:
Nit: add comment in the code about it